### DISCUSSION TITLE
Multiple instances of virtual cam in one computer

### DISCUSSION THREAD

**Discussion by SoberNov** [letmaik/pyvirtualcam#64] [Q&A]
I can set up one virtual camera using obs thru pyvirtual cam import, and i am using recorded video to cast the video to the virtual cam and try to get zoom to see the virtual cam. I am wondering is it possible to create multiple instances of virtual camera in one computer and also allow zoom to recognize different video source?

**Comment by letmaik** [comment]
Yes that's possible. I'm assuming you're on Windows. OBS only has a single virtual camera, but there's another backend available called Unity Capture, see https://github.com/letmaik/pyvirtualcam#windows-unity-capture. See the API docs on how to select specific virtual cameras: https://letmaik.github.io/pyvirtualcam/
